caput/mpiutil.py

```diff
@@ -677,6 +679,60 @@ def parallel_rows_write_hdf5(fname, dsetname, local_data, shape, comm=_comm):
     lock_and_write_buffer(local_data, fname, offset + sr * nc, lr * nc)
 
 
+# Need to disable the pylint error here as we only need to implement one method
+class MPILogFilter(logging.Filter):  # pylint: disable=too-few-public-methods
```

Comment:
Will this replace the one in draco?

Reply:
Yup. When I've got this merged I'll remove it from draco.